private_sharing/templates/private_sharing/remove-project-members.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'panel.html' %}
+
+{% load bootstrap_tags %}
+{% load static from staticfiles %}
+
+{% block head_title %}{{ aoeueoa }}Remove project members{% endblock %}
+
+{% block panel_content %}
+
+<h4>Project: {{object.name}}</h4>
+
+<p>
+  <b>CAUTION: this action cannot be reversed by you!</b><br>
```

Comment:
Besides maybe going into the db and manually re-adding, I don't quite see an easy way to reverse this?  Maybe have something in the admin page?

Reply:
Another way to reverse it is that the individual can re-join... I hope projects don't screw up in removals, but if they do, yeah I guess they could petition us to do something like you describe. I don't envision it coming up (and hopefully never), so wouldn't worry about making an easy tool for doing it...